server/controllers/routes.js

```diff
@@ -7,5 +7,54 @@ routers.get('/', (req, res) => {
   res.send('main page layout');
 });
 
+routers.post('/create-user', (req, res) => {
+  // check the database if user exists
+
+  // create new entry in database with parent's info
+
+  // direct them to /parent-sign-in, and direct to /parent-profile
+  res.redirect('/parent-profile');
+});
+
+routers.post('/create-user', (req, res) => {
+  // check the database if user exists
+
+  // create new entry in database with parent's info
+
+  // direct them to /parent-sign-in, and direct to /parent-profile
+  res.redirect('/parent-profile');
```

Comment:
use React router instead, in the front end to navigate between routes.
This redirect will cause you conflict's when you serve the built public file